cmd/crowdsec/main.go

```diff
@@ -295,9 +297,45 @@ func LoadConfig(cConfig *csconfig.Config) error {
 		return err
 	}
 
+	err := LoadFeatureFlags(cConfig, log.StandardLogger())
+	if err != nil {
+		return err
+	}
+
 	return nil
 }
 
+
+// LoadFeatureFlags parses {ConfigDir}/feature.yaml to enable/disable features.
+//
+// Since CROWDSEC_FEATURE_ envvars are parsed before config.yaml,
+// when the logger is not yet initialized, we also log here a recap
+// of what has been enabled.
+func LoadFeatureFlags(cConfig *csconfig.Config, logger *log.Logger) error {
+	featurePath := filepath.Join(cConfig.ConfigPaths.ConfigDir, "feature.yaml")
+
+	if err := fflag.CrowdsecFeatures.SetFromYamlFile(featurePath, logger); err != nil {
+		return fmt.Errorf("file %s: %s", featurePath, err)
+	}
+
+	enabledFeatures := []string{}
+
+	featureStatus, err := fflag.CrowdsecFeatures.GetFeatureStatus()
+	if err != nil {
+		return fmt.Errorf("unable to get feature status: %s", err)
+	}
+
+	for feature, enabled := range featureStatus {
+		if enabled {
+			enabledFeatures = append(enabledFeatures, feature)
+		}
+	}
+	logger.Infof("Enabled features: %s", strings.Join(enabledFeatures, ", "))
```

Comment:
maybe only log if at least one feature was enabled ?

Reply:
yes I forgot to put the <none>. I prefer to be explicit